Say/ViewController.swift

```diff
@@ -57,6 +57,8 @@ class ViewController: NSViewController {
     let voiceSavePanel = NSSavePanel()
     /// Open panel for "Open" menu
     let textOpenPanel = NSOpenPanel()
+    
+    var sayObj: Say! = nil;
```

Comment:
I think instance variable is not nessessary here. Swift will capture the variable into the dispatch qeueu blocks.